### DISCUSSION TITLE
Example for MSD with SDIO SD card interface

### DISCUSSION THREAD

**Discussion by wbadry** [adafruit/Adafruit_TinyUSB_Arduino#401] [General]
Hi,
Is there any working example for the ESP32S3 with an SDIO SD card interface as a USB mass storage device?

The examples seem to only support the SPI interface.

**Comment by zachtos** [comment]
I would love this to work for the ESP32-S3.  I love the USB-MSD to add files over USB to the SD card.  My ultimate wishes would be that it could work in SDIO 1 bit and 4 bit mode with this library.   Bonus if anyone figures out how to update firmware by dragging a bin file into the USB-MSD!  We would pay for this.
